notebooks/05-operations/01-druid-logs-configuration.ipynb

```diff
@@ -0,0 +1,524 @@
+    "\n",
+    "* [`Properties`](https://logging.apache.org/log4j/2.x/manual/configuration.html#PropertySubstitution) provide key/values pairs that may be used throughout the configuration file.\n",
+    "* [`Appenders`](https://logging.apache.org/log4j/2.x/manual/appenders.html) designate the format of log messages and determine the target for the messages.\n",
+    "* [`Loggers`](https://logging.apache.org/log4j/2.x/manual/configuration.html#Loggers) filter the log messages and dispense them to Appenders. Loggers can filter messages based on the Java package and/or class and by message priority."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ca62693b-50f4-4831-a092-dbc8e06d8191",
+   "metadata": {},
+   "source": [
+    "## Properties\n",
+    "\n",
+    "In Druid, `Properties` are leveraged to set a location for all logs, giving Druid the location for log files at start-up.\n",
+    "\n",
+    "By default, this location is a \"log\" folder at the root of your Druid installation, but [can be over-ridden]((https://druid.apache.org/docs/latest/configuration/logging#log-directory) by using the `DRUID_LOG_DIR` system variable."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bec19737-2773-4c66-a05e-d0b1a707c4c8",
+   "metadata": {},
+   "source": [
+    "## Appenders\n",
+    "\n",
+    "There are two `appenders`:\n",
+    "\n",
+    "* \"Console\", which is a a [`Console`](https://logging.apache.org/log4j/log4j-2.4/manual/appenders.html#ConsoleAppender) appender for `SYSTEM_OUT`.\n",
+    "* \"FileAppender\", which is a [`RollingRandomAccessFile`](https://logging.apache.org/log4j/log4j-2.4/manual/appenders.html#RollingRandomAccessFileAppender) appender for detailed process logs.\n",
+    "\n",
+    "### Start a Druid instance\n",
+    "\n",
+    "The default Configuration for Druid does not include a [`monitorInterval`](https://logging.apache.org/log4j/log4j-2.4/manual/configuration.html#AutomaticReconfiguration) property, so changes to logging configuration are only recognised when a process restarts.\n",
+    "\n",
+    "For the purposes of this tutorial, run the following command to add a `monitorInterval` property to the Configuration.\n",
+    "\n",
+    "```bash\n",
+    "sed -i '' 's/<Configuration status=\"WARN\">/<Configuration status=\"WARN\" monitorInterval=\"5\">/' \\\n",
+    "  ~/learn-druid-local/apache-druid/conf/druid/auto/_common/log4j2.xml\n",
+    "```\n",
+    "\n",
+    "Start Druid with the following command:\n",
+    "\n",
+    "```bash\n",
+    "nohup ~/learn-druid-local/apache-druid/bin/start-druid & disown > log.out 2> log.err < /dev/null\n",
+    "```\n",
+    "\n",
+    "### Look at the standard log files\n",
+    "\n",
+    "Since Druid is a distributed system, we will find log files for each Druid process. In addition, Druid also captures the output written to the standard output. Use the following command to take a look at what is here:\n",
```

Comment:
```suggestion
    "Since Druid is a distributed system, it contains log files for each Druid process. In addition, Druid also captures the output written to the standard output. Use the following command to take a look at what is here:\n",
```